### PR TITLE
added logOut functional

### PR DESCRIPTION
**Pull request by Rashevskyyy**

### CODE REVIEW

File: client/src/Store/chat/saga.js [hunk 1]
```diff
@@ -21,13 +21,14 @@ import {
     putNewRoom,
     sendUsersRequest,
     reciveSuccessUsersRequest,
-    reciveErrorUsersRequest,
+    reciveErrorUsersRequest, setValue,
```
Comment (MaxShvachko) [inline client/src/Store/chat/saga.js]: move to next line

File: client/src/Store/chat/saga.js [hunk 2]
```diff
@@ -21,13 +21,14 @@ import {
...
 } from './actions';
 import { userInfo } from '../user/selectors';
 import { newMessage, currentRoom, messages } from './selectors';
 import { getRequest, postRequest } from '../../helpers/requests';
 import { routes } from '../../constants/routes';
 import { support } from '/src/helpers/support';
+import {setAuthValues} from "../user/actions";
```
Comment (MaxShvachko) [inline client/src/Store/chat/saga.js]: add spaces after and before setAuthValues

File: client/src/components/Header/HeaderControlPanel/HeaderControlPanel.jsx [hunk 3]
```diff
@@ -19,10 +19,16 @@ const HeaderControlPanel = ({ themeMode, setValue, history, location }) => {
         support.setSessionStorageItem('themeMode', target.value);
         setValue({ name: 'themeMode', value: target.value });
     };
+    const handleLogOutClick = () => {
+        logOut();
+        history.push(APP_ROUTES.login);
+        console.log(history)
```
Comment (MaxShvachko) [inline client/src/components/Header/HeaderControlPanel/HeaderControlPanel.jsx]: console log delete

File: server/Classes/SocketMaster.js [hunk 4]
```diff
@@ -11,6 +11,7 @@ class SocketMaster extends ChatManager {
         this.io.on('connection', (socket) => {
             console.log('user connected', socket.id);
             socket.on('disconnect', () => {
+                console.log('disconnect')
```
Comment (MaxShvachko) [inline server/Classes/SocketMaster.js:14]: log delete

Comment (SkrypnikMR) [reply server/Classes/SocketMaster.js:14]: this is the socket logging ))

Comment (Rashevskyyy) [reply server/Classes/SocketMaster.js:14]: +, I think you need it

**Pull request merged**